scripts/minikube.sh

```diff
@@ -5,7 +5,7 @@
 function wait_for_ssh() {
     local tries=100
     while ((tries > 0)); do
-        if minikube ssh echo connected &>/dev/null; then
+        if /usr/local/bin/minikube ssh echo connected &>/dev/null; then
```

Comment:
minikube won't be installed always in this path isn't it?

Reply:
That is where this script installs it. It is just that `/usr/local/bin` is not in the `PATH` for root on CentOS, so it will not be found, even after installing it there.

Reply:
will it cause any issue if the minikube is already present and set in PATH for a different location?

Reply:
no, I do not think so

Reply:
@nixpanic if we run from our system the path may be different. Hard ocding the path makes it difficult Isnt it ?

Reply:
This seems problematic if the minikube is already present in  `/usr/bin/minikube` with current changes it tries to install at `/usr/local/bin` even if the minikube is already present.

Reply:
Well, the script already installed minikube in `/usr/local/bin`, I did not want to change that. I could add detection to see if a minikube executable is available, and use that instead. And then have it only install under `/usr/local/bin` if there is no other minikube executable.

Is that something you prefer?

Reply:
This has been done now, all minikube calls use `${minikube}` which gets detected on start of the script.